aci/resource_aci_cloudctxprofile.go

```diff
@@ -232,6 +256,13 @@ func resourceAciCloudContextProfileCreate(ctx context.Context, d *schema.Resourc
 		checkDns = append(checkDns, temp.(string))
 	}
 
+	accessPolicy := ""
+	if tmpVar, ok := d.GetOk("access_policy_type"); ok {
+		accessPolicy = tmpVar.(string)
+		accessDn := fmt.Sprintf("uni/tn-infra/accesspolicy-%s", accessPolicy)
```

Comment:
```suggestion
		accessDn := fmt.Sprintf("uni/tn-infra/accesspolicy-%s", tmpVar.(string))
```

Reply:
accessPolicy will be used later so that we cannot omit it here